Source/santad/Logs/SNTSyslogEventLog.m

```diff
@@ -169,8 +169,8 @@ - (void)logExecution:(santa_message_t)message withDecision:(SNTCachedDecision *)
       mode = @"U"; break;
   }
 
-  [outLog appendFormat:@"|pid=%d|ppid=%d|uid=%d|user=%@|gid=%d|group=%@|mode=%@|path=%@",
-      message.pid, message.ppid,
+  [outLog appendFormat:@"|pid=%d|pidversion=%d|ppid=%d|uid=%d|user=%@|gid=%d|group=%@|mode=%@|path=%@",
```

Comment:
Probably worth adding this to `logFork:` and `logExit:` near the bottom of the file and `logFileModification:` near the top.